### ISSUE TITLE
Suggestion to rename the mod now that any sound can be silenced

### ISSUE THREAD

**Issue by SnowShock35** [kreezxil/BadWitherNoCookie#1]
Hi there,

I've been following the development of the mod pretty much since you forked it from the former author and I'm so glad you have maintained and even added so many new features. With the newest version I see that any sounds can now be silenced so it seemed like a good idea to suggest a name change. Obviously this can be a little bit of a hassle to do however, I think the name bad wither no cookie doesn't really fit the purpose of the mod now. Even if it was to have a name change for the next Minecraft version I do think a name change would be appropriate given the mods newer capabilities.

I understand if you don't agree the same - probably just me that thinks this. Just thought I would bring it up just to see what you opinion on the subject is. 

Many thanks for all your hard work.
:heart:

**Comment by kreezxil** [comment]
Did you have a name in mind?

I'm thinking I could use "TheSilence" but that might be construed as a Dr. Who reference. 
So maybe "SilenceIt" or "SilenceAll" or simply "Silence".

Also I wouldn't change the name of the current project, I would make a new mod project called w/e the new name is and have link on the BWNCR pointing to the new project.

Or I can leave it alone, because if you don't use the added feature, it is still a wither silencer.

**Comment by kreezxil** [comment]
I think I got it.

"Silence is Golden"

anyhow what do you search for when you want silence a sound via a mod?

How did you discover Bad Wither No Cookie in the first place?

**Comment by SnowShock35** [comment]
Hmm not too sure in regards to names. Also, idea of just creating a new project from scratch and manually migrating the code is probably the best bet. However, CurseForge projects are easily changed with renaming it in the settings and they'll force redirect from old links to the new ones - be sure to change both name and slug though. 

As for how I found it I can't quite remember how I found Bad Wither No Cookie. But I found yours when I looked at forked branches of their version. 

Something like `Global Mute` or `Hush Sound`s or `Hush Wither` or something that's not plain but clear what it does. I usually use synonyms of the plain-English terms to get a good name. See if that helps spark ideas and ask a few others too :smile:
